subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/javaapplication/multi/app/App.java.template

```diff
@@ -8,10 +8,13 @@ import static ${basePackageName.raw}.utilities.StringUtils.join;
 import static ${basePackageName.raw}.utilities.StringUtils.split;
 import static ${basePackageName.raw}.app.MessageUtils.getMessage;
 
+import org.apache.commons.text.WordUtils;
+
 public class App {
     public static void main(String[] args) {
         LinkedList tokens;
         tokens = split(getMessage());
-        System.out.println(join(tokens));
+        String result = join(tokens);
+        System.out.println(WordUtils.capitalize(result));
```

Comment:
:x: There is also a "App.scala.template" and "App.kotlin.template" that should be adjusted in the same way.